uaclient/cli.py

```diff
@@ -226,30 +226,25 @@ def attach_parser(parser):
 def fix_parser(parser):
     """Build or extend an arg parser for fix subcommand."""
     parser.usage = USAGE_TMPL.format(
-        name=NAME, command="fix --beta <CVE-yyyy-nnnn+>|<USN-nnnn-d+>"
+        name=NAME, command="fix <CVE-yyyy-nnnn>|<USN-nnnn>"
     )
     parser.prog = "fix"
     parser._optionals.title = "Flags"
-    parser.add_argument(
-        "--beta",
-        required=True,
-        action="store_true",
-        help="allow using this beta command",
-    )
     parser.add_argument(
         "security_issue",
         help=(
             "Security vulnerability ID to inspect and resolve on this system."
-            " Format: CVE-yyyy-nnnn, CVE-yyyy-nnnnnnn or USN-nnnn-dd"
+            " Format: CVE-yyyy-nnnn or USN-nnnn"
         ),
     )
     return parser
 
 
 def action_fix(args, cfg, **kwargs):
     if not re.match(security.CVE_OR_USN_REGEX, args.security_issue):
+        # TODO(review of error messaging to reduce output)
```

Comment:
I've not reviewed how we handle todo in the codebase. I want to raise the question on it's cool to have a todo, but in the past I've liked projects that the todo referenced a bug/issue number that could provide more context, discussion, and progress indication.

Reply:
Good suggestion here and we should make the TODO() reference an issue. Turns out, this commit wasn't a clean revert, I shouldn't have included that comment in this as Lucas had addressed that point when the former PR was in WIP status. I've reapplied the proper diff which included his review comments. So this comment goes away.